packages/lab/stories/overlay/overlay.stories.tsx

```diff
@@ -55,24 +55,19 @@ const OverlayTemplate = (props: OverlayProps) => {
   );
 };
 
-export const Default = (props: OverlayProps) => {
-  return OverlayTemplate({ id: "overlay-default", ...props });
-};
-
-export const Bottom = (props: OverlayProps) => {
-  return OverlayTemplate({
+export const Bottom = () => {
```

Comment:
Is there a reason why we aren't using the standard template implementation e.g. https://storybook.js.org/docs/7.6/api/csf#upgrading-from-csf-2-to-csf-3